test/config.spec.js

```diff
@@ -114,7 +114,7 @@ describe('Config', () => {
       enabled: false,
       debug: true,
       hostname: 'agent',
-      overrideUrl: 'https://agent2:443',
+      overrideUrl: 'https://agent2:7777',
```

Comment:
Why not just `url` or `endpoint`?

Reply:
Yeah, i thought it might be a little confusing with just `url`, but the update docs should clear that up incase its not clear